composer.json

```diff
@@ -21,17 +21,17 @@
         "psr/container": "^1.0",
         "psr/event-dispatcher": "^1.0",
         "symfony/contracts": "^1.1 || ^2.0",
-        "symfony/expression-language": "^3.4 || ^4.0 || ^5.0",
-        "symfony/framework-bundle":   "^3.4 || ^4.0 || ^5.0",
-        "symfony/property-access": "^3.4 || ^4.0 || ^5.0",
-        "symfony/yaml":   "^3.4 || ^4.0 || ^5.0"
+        "symfony/expression-language": "^3.4 || ^4.0 || ^5.0 || ^6.0",
```

Comment:
same [comment for very old symfony versions](https://github.com/BedrockStreaming/GuzzleHttpBundle/pull/75#discussion_r761796423)

Reply:
Like the other PR, I've removed all unmaintained Symfony versions.